amf-validation/shared/src/main/scala/amf/validation/internal/shacl/custom/CustomShaclValidator.scala

```diff
@@ -95,29 +95,41 @@ class CustomShaclValidator(customFunctions: CustomShaclFunctions, options: Shacl
     }
   }
 
+  private def processPlainElement(element: AmfElement): Seq[(AmfElement, Option[Any])] = {
```

Comment:
This tuple can be extracted to a case class since it's used as a return type of many functions